website/docs/contributing.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 3
+---
+
+# How to contribute to FerretDB
+
+FerretDB is an open source project and everyone – developers and non-developers – is welcome to contribute.
+If you’re interested in contributing to FerretDB projects, this documentation will help you get started.
+
+Here are some of the current FerretDB projects that you can fork and contribute to:
+
+* [FerretDB](https://github.com/FerretDB/FerretDB): This repository contains the FerretDB project – an open source alternative project to MongoDB.
+* [github-actions](https://github.com/FerretDB/github-actions): This repository houses our shared GitHub Actions for FerretDB, dance, and other repositories.
+* [dance (stub)](https://github.com/FerretDB/dance): This repository is a FerretDB integration testing tool.
+
+For those taking their first steps in contributing to an open source project.
+Please take a look at this post on [how to contribute to open source software](https://www.ferretdb.io/how-to-contribute-to-open-source-2022/).
+
+## Get Started
+
+You don’t have to be a developer to contribute to FerretDB projects, you can even get started by helping us improve this documentation.
+If you have any questions or suggestions on how we can improve, kindly join our [Slack chat](https://github.com/FerretDB/FerretDB/discussions) or [GitHub discussions](https://join.slack.com/t/ferretdb/shared_invite/zt-zqe9hj8g-ZcMG3~5Cs5u9uuOPnZB8~A).
```

Comment:
Could we link to `/intro/#community` instead? I'm a bit worried about that Slack URL that should be updated from time to time being in too many places